monolithe/generators/lang/python/converter.py

```diff
@@ -37,7 +37,9 @@ def _string_clean(string):
         "vCenter": "Vcenter",
         "VPort": "Vport",
         "IPv6": "Ipv6",
-        "IPv4": "Ipv4"
+        "IPv4": "Ipv4",
+        "DHCPv4": "DhcpV4",
```

Comment:
so here we have capital V while for ipvx we have lowercase v , and @brncoguz explained me that also the API definition is holding inconsistency ; I believe we should fix that

Reply:
you're right, it should be enable_dhcpv4 and enable_dhcpv6, I fixed the PR.